activities/TamTamMicro.activity/piano.js

```diff
@@ -69,6 +98,16 @@ enyo.kind({
 		player.connect(pitchShift);
 		player.autostart = true;
 	},
+	
+	create: function() {
+		this.inherited(arguments);
+		this.collection = 0;
+		var that = this;
+		document.addEventListener('keydown', function(event) {
```

Comment:
Be careful: each time the screen is created, a new event listener is registered. You should remove the event listener when leaving the piano mode else the method will be called one more time each time you reopen the screen.